matcher/service/openreview_interface.py

```diff
@@ -67,6 +72,12 @@ def validate_score_spec(self):
                 raise error_handle
 
     def validate_group(self, group_id):
+        if self._task == "deploy":
```

Comment:
given that we may have more tasks in the future, can we run this validation only when there is '/match' call? Instead of passing a task parameter, can you call this function from the match router or from the Matcher class?